src/libs/EmojiUtils.tsx

```diff
@@ -436,17 +436,6 @@ function suggestEmojis(text: string, locale: Locale = CONST.LOCALES.DEFAULT, lim
     return lodashSortBy(matching, (emoji) => sortByName(emoji, emojiData));
 }
 
-/**
- * Retrieve preferredSkinTone as Number to prevent legacy 'default' String value
- */
-const getPreferredSkinToneIndex = (value: OnyxEntry<string | number>): number => {
-    if (value !== null && Number.isInteger(Number(value))) {
-        return Number(value);
-    }
-
-    return CONST.EMOJI_DEFAULT_SKIN_TONE;
-};
-
```

Comment:
Leaving a note here: There is no user with a string value in the database, so we removed it.